utils/onnxmlir/VERSION_NUMBER

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Version number here is different than `onnx-mlir/VERSION_NUMBER`. Is that on purpose?
Also the number here is different than the version listed in the file below.

Reply:
Fixed. I used a version number different from the onnx-mlir. The version for onnxmlir is for upstream onnxmlir to pip repo. It could be viewed as a pre-required package.

Reply:
May I suggest a `0.1.0` and move to a `1.0.0` once we have the installation of the compiler with it at least one z?